project.clj

```diff
@@ -95,7 +95,7 @@
   :cljsbuild {
     :builds [
       {:id "dev"
-       :source-paths ["src/clj/game/core" "src/cljs/nr" "src/cljs/dev" "src/cljc"]
+       :source-paths ["src/clj/game/core" "src/cljs/nr" "src/cljs/nr/gameboard" "src/cljs/dev" "src/cljc"]
```

Comment:
Is this needed? I think :source-paths includes all subdirectories.

Reply:
Oh seems you're right.